ports/esp32s2/supervisor/port.c

```diff
@@ -54,6 +55,7 @@
 
 uint32_t* heap;
 uint32_t heap_size;
+extern TaskHandle_t xTaskToNotify;
```

Comment:
Could this go in `ports/esp32s2/supervisor/port.h`?

```suggestion
extern TaskHandle_t sleeping_circuitpython_task;
```